OpenRA.WindowsLauncher/OpenRA.WindowsLauncher.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UseWindowsForms>true</UseWindowsForms>
+    <OutputPath>..</OutputPath>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <ApplicationIcon>$(icon)</ApplicationIcon>
+    <CodeAnalysisRuleSet>..\OpenRA.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\OpenRA.Game\OpenRA.Game.csproj" />
+    <AdditionalFiles Include="../stylecop.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="TiberianDawn.ico" />
```

Comment:
Can we reference these from the packaging dir instead of duplicating them into the WindowsLauncher directory?

Reply:
We dont need them in packaging dir  anymore.

Reply:
I was hoping to eventually remove all of our platform-specific artwork files from the repository, and build the ico and icns files at package time from a common set of png files. I set this up for the mod sdk in https://github.com/OpenRA/OpenRAModSDK/pull/144.